packages/ifpack2/src/Ifpack2_BlockTriDiContainer_impl.hpp

```diff
@@ -55,6 +55,8 @@
 #include <KokkosBatched_Copy_Impl.hpp>
 #include <KokkosBatched_AddRadial_Decl.hpp>
 #include <KokkosBatched_AddRadial_Impl.hpp>
+#include <KokkosBatched_SetIdentity_Decl.hpp>
```

Comment:
There's a pattern here of including two header files.  I wonder if it would make sense for kokkos-kernels to generate header files like `KokkosBatched_SetIdentity.hpp`, that include both the Decl and Impl header files.

Reply:
I was ambitious that sometime I can do eti of these files (too much ambitious). I can just create a single header file to include all files. something like e.g. KokkosBatched_FunctorLevelFunctions.hpp